passgen/random/build.gradle.kts

```diff
@@ -0,0 +1,4 @@
+plugins {
+  kotlin("jvm")
```

Comment:
Leaving a note for myself to apply this plugin in the `kotlin-library` convention plugin